generators/client/templates/angular/src/main/webapp/app/shared/_shared-libs.module.ts

```diff
@@ -9,12 +9,13 @@ import { InfiniteScrollModule } from 'angular2-infinite-scroll';
 @NgModule({
     imports: [
         NgbModule.forRoot(),
-        NgJhipsterModule.forRoot({
+        // TODO uncomment the code below and make it works with AOT
+        NgJhipsterModule/*.forRoot({
```

Comment:
No I have no idea. To reproduce it, clone my repo, switch to this branch, generate an app, uncomment the code and try to run `yarn run ngc`.